Adding troubleshooting info about full disk access (#7938)
fleetdm/fleet: docs/Using-Fleet/Adding-hosts.md

SELECT * FROM file WHERE path LIKE '/Users/%/Downloads/%%';
```

> If this query returns files, the profile was applied, as *Downloads* is a

Reviewer: ```suggestion
If this query returns files, the profile was applied, as **Downloads** is a
```
Please avoid using italics for emphasis. Use bold instead.

protected location. You can now enjoy the benefits of osquery on all system files and start

> using the *es_process_events* table!

Reviewer: ```suggestion
using the **es_process_events** table!
```

> If this query does not return data, you can look at operating system logs to confirm that full disk

Reviewer: ```suggestion
If this query does not return data, you can look at operating system logs to confirm whether or not full disk
```

access is not applied.

See the last hour of logs related to TCC permissions with this command:

`log show --predicate 'subsystem == "com.apple.TCC"' --info --last 1h`

> You can then look for `orbit` or `osquery` to narrow down results.

Reviewer 2: What does "narrow down results" mean? Or more generally, ok I got the logs and maybe I've searched for "orbit" or "osquery" to find relevant ones. What should I expect to see? What do I do in response to things I might see? Could be as simple as link them to something above, but I think we should give the reader next steps here.

Author: I don't have logs I can use for this now. Next time we are playing with our full disk access I could add real examples.
